src/apps/search-result/search-result.tsx

```diff
@@ -124,6 +133,32 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
 
   const worksAreLoaded = Boolean(resultItems.length);
 
+  useEffect(() => {
+    if (isFirstLoad) {
+      setIsFirstLoad(false);
+      return;
+    }
```

Comment:
Please document why this is needed.

Do we not want to track the number of results for the initial search result?

Reply:
well no, because the initial search result is always 0. It is set on line 48 with ` const [hitcount, setHitCount] = useState<number>(0);`

Reply:
OK. Have you considering making the default value for the `hitcount` `null` instead of `0` and then checking for that instead? To me it seems preferable to adding another state.

Reply:
That sounds very reasonable! Haven't thought of that, but now that's the way we do it :)